### ISSUE TITLE
fix: aria label misuse

### ISSUE THREAD

**Issue by dargmuesli** [harlan-zw/nuxt-seo-kit#33]
### Describe the bug

https://github.com/harlan-zw/nuxt-seo-kit/blob/946db73cc73f4c43e8adc142f4f7ec4edaeb78b3/layer/components/Breadcrumbs.vue#L15
sets `aria-label="Breadcrumb"` but github.com/nuxt-modules/html-validator reports an aria label misuse (https://rocketvalidator.com/html-validation/possible-misuse-of-aria-label-if-you-disagree-with-this-warning-file-an-issue-report-or-send-e-mail-to-www-validator-w3-org).

### Reproduction

will create one if description is not clear already

### System Info

```Shell
-
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/harlan-zw/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by harlan-zw** [comment]
Hey, thanks for the issue :)

I think it the aria-label should be on a `nav` wrapping the ul, I'll sort that out soon
